lib/semian.rb

```diff
@@ -296,7 +296,19 @@ def require_keys!(required = [], **options)
   end
 end
 
+def hostname
+  v = %w(KUBE_HOSTNAME KUBE_HOST_NAME KUBE_NODENAME KUBE_NODE_NAME NODENAME NODE_NAME HOSTNAME HOST_NAME)
+  var = v.filter { |x| ENV.include?(x) }.first
```

Comment:
`filter { ... }.first` is the same as `find { ... }`, minus the need to construct the intermediate array.

Reply:
Nice!